src/components/list-facet/index.js

```diff
@@ -178,31 +187,41 @@ class FederatedListFacet extends React.Component {
       const listFacetHierarchyLis = [];
       // Define array of checkbox Lis which we'll populate with react fragments, per type.
       const listFacetHierarchyTermsLis = [];
+
+      /* eslint-disable react/no-array-index-key */
+
       // Iterate through types (accordion lis).
```

Comment:
I wasn't sure how to refactor this section to not use loop indexes in keys, so I temporarily disabled the rule. We may want to revisit this section later, so a `TODO` might be in order here.